SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -97,56 +162,26 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 			OuterObjectRef = PackageMap->GetUnrealObjectRefFromNetGUID(NetGUID);
 		}
 
-		// No path in SpatialOS should contain a PIE prefix.
+#if DO_GUARD_SLOW
```

Comment:
We sure this is particularly slow? I'm more in favour of being verbose about this error in development, and potentially in a subsequent release removing the check, or reducing it to a slow_check.

Reply:
I could make it run all the time, and create a ticket for removing this if it's not hit?